ga4gh/client.py

```diff
@@ -16,12 +16,37 @@
 
 
 class HTTPClient(object):
-
-    workaroundGoogle = 'google'
-
     """
-    Simple HTTP client for the GA4GH protocol.
+    GA4GH Http Client
     """
+    workaroundGoogle = 'google'
+
+    class RunRequestContainer(object):
+        """
+        A container class for arguments to runRequest
+        """
+        def __init__(
+            self, request=None, url=None, protocolClass=None,
+                listAttr=None, requestMethod='post', id_=None):
+            self.request = request
+            self.url = url
+            self.protocolClass = protocolClass
+            self.listAttr = listAttr
+            self.requestMethod = requestMethod
+            self.id_ = id_
```

Comment:
There's a tricky style issue here. Should we have `self.id_` or `self.id`? Having an `id` attribute on an object is fine as far as I know (and we're doing it plenty of other places), so I'd probably prefer `self.id`. Definitely variables should be called `id_` though.